examples/pruner/README.md

```diff
@@ -0,0 +1,29 @@
+# Pruning images using CronJob
+
+This example shows an image pruning happening in an automated fashion using the Kubernetes CronJobs that
+are available in OpenShift Origin from version 3.5.
```

Comment:
they exists from 3.5 I believe,   but they are usable for this from 3.6 ;-) maybe i will delete this sentence :)